components/camera/transformpipeline/pipeline.go

```diff
@@ -60,7 +60,6 @@ func init() {
 
 // transformConfig specifies a stream and list of transforms to apply on images/pointclouds coming from a source camera.
 type transformConfig struct {
-	resource.TriviallyValidateConfig
```

Comment:
this was weird. We had a double implementation of Vlaidate in pipeline, so I don't know which one it could have called, if it was deterministic at all.

Reply:
Golang moment